doc/source/serve/ml-models.rst

```diff
@@ -73,6 +74,34 @@ That's it. Let's take a look at an example:
 
 .. literalinclude:: ../../../python/ray/serve/examples/doc/snippet_model_composition.py
 
+.. _serve-model-ensemble:
+
+Model Ensemble
+=================
+
+Ray Serve supports creating different ensemble models
+
+To define an ensemble of different models you need to do three things:
+
+1. Define your underlying sub models (the ones that make up the ensemble) as
+   Ray Serve deployments.
+2. Define your ensemble model, using the handles of the underlying models
+   (see the example below).
+3. Define a deployment representing this ensemble model and query it!
+
+In order to avoid synchronous execution in the ensemble model, you'll need to make
+the function asynchronous by using an ``async def``. In contrast to a composition model,
+within an ensemble model, you want to call **all** sub models in parallel. This will be
+achieved by sending all prediction calls to the sub models via async by using
+``asyncio.wait()``. If you want to scale your ensemble model, make sure to spawn
+multiple sub model deployment replicas, so that every call to the higher level
+ensemble model can be run in parallel by using ``num_replicas=5``.
```

Comment:
how about we phrase it as something like "Each serve deployment used in an ensemble use case is independently scalable via changing `num_replicas`" ? We can use 5 as example in code but don't have to surface it in the documentation.

Reply:
yes sure added it.